slsim/Plots/lens_plots.py

```diff
@@ -36,31 +39,70 @@ def rgb_image(self, lens_class, rgb_band_list, add_noise=True):
         :param add_noise: boolean flag, set to True to add noise to the image, default
             is True
         """
-        image_r = simulate_image(
-            lens_class=lens_class,
-            band=rgb_band_list[0],
-            num_pix=self.num_pix,
-            add_noise=add_noise,
-            observatory=self._observatory,
-            **self._kwargs
-        )
-        image_g = simulate_image(
-            lens_class=lens_class,
-            band=rgb_band_list[1],
-            num_pix=self.num_pix,
-            add_noise=add_noise,
-            observatory=self._observatory,
-            **self._kwargs
-        )
-        image_b = simulate_image(
-            lens_class=lens_class,
-            band=rgb_band_list[2],
-            num_pix=self.num_pix,
-            add_noise=add_noise,
-            observatory=self._observatory,
-            **self._kwargs
-        )
-        image_rgb = make_lupton_rgb(image_r, image_g, image_b, stretch=0.5)
+        if self._observatory == "Roman":
```

Comment:
maybe all these inputs can be shortened if the simulate_roman_image() and simulate_image() follow exactly the same API? Then we only have to overwrite/define the function to be called as an if/else statement and everything else is identical

Reply:
I can try to shorten this part of the code